.storybook/preview.tsx

```diff
@@ -29,8 +29,8 @@ export const decorators = [
   Story => (
     <SiteProvider>
       <LocaleProvider>
-        <ThemeProvider font={openSans.style.fontFamily}>
-          <div data-test-id="story-root">
+        <ThemeProvider>
```

Comment:
This is not what it was asked.

The update should happen within `ThemeProvider` and should remove the `<style>` that is being defined there.

And then a `className` with `openSans.variable` (NOT `openSans.className`)` should be added.